.prettierignore

```diff
@@ -0,0 +1,6 @@
+src/svgs/deprecated/index.js
```

Comment:
I'm okay with prettier changing the formatting for these files--does it interfere with `// @create-index`?

Reply:
- ok, so this is generated and used only by storybook (may need to look at possibility to use icons from libs in other PRs althought build is not required I believe for storybook).  
- So for example take the `trailingComma` format it gets auto-formatted when you do a push
- Then you are working locally in a total unrelated subject and if you run storybook it will change the files back to no commas because it is autogenerated, Then you have 5 files in GIT to add. This is infinite loop so I decided to just not autoformat these files since they are autogenerated

- Now looking at these again I think we can even gitignore them

Reply:
OK, so I tried putting them in `.gitignore` but that crahsed the storybook build since it uses `create-index` in update mode and couldnt find the index files to update. We may try again removing the update flag but I just decided to leave it in prettierignore for this PR since I dont want more changes to build system here